OMCompiler/Compiler/SimCode/SimCodeUtil.mo

```diff
@@ -9319,7 +9324,12 @@ algorithm
         isValueChangeable = BackendVariable.varHasConstantStartExp(dlowVar);
         caus = getCausality(dlowVar, vars, isValueChangeable);
         variability = SimCodeVar.CONTINUOUS(); // state() should be CONTINUOUS
-        initial_ = setInitialAttribute(dlowVar, variability, caus, isFixed, iterationVars);
+        //check for Alias vars, if Alias vars then set initial = CALCULATED(), as only reference variable should have start value, according to FmiSpecification2.0
+        if valueEq(aliasvar, SimCodeVar.NOALIAS()) and not BackendVariable.isConst(dlowVar) then
+          initial_ = setInitialAttribute(dlowVar, variability, caus, isFixed, iterationVars);
+        else
+          initial_ = SimCodeVar.CALCULATED();
+        end if;
```

Comment:
This repeating pattern should be moved inside the function `setInitialAttribute` to avoid redundant code.

Reply:
Yes, but doing it in setInitialAttribute function does more work, when we already knew that aliasVars set "initial = calculated"  But then we avoid redudant code